deploy/prod.yml

```diff
@@ -16,6 +16,10 @@ accessOutbound:
     - data.brreg.no
     - g.nav.no
     - idporten.no
+accessInbound:
```

Comment:
skal ikke tokenx-generator legges til for prod også?

Reply:
Nei det er bare for å kunne genererte tokens i dev.